providers/implementations/kdfs/hkdf.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016-2022 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2016-2023 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
This is updated automatically when the release happens, so isn't necessary.
It's harmless, so leave it.